tests/e2e/hellodapr/hellodapr_test.go

```diff
@@ -56,6 +57,7 @@ func TestMain(m *testing.M) {
 			AppName:           "hellobluedapr",
 			DaprEnabled:       true,
 			ImageName:         "e2e-hellodapr",
+			SidecarImage:      runtime_utils.GetEnvOrElse("DAPR_TEST_OLD_IMAGE", ""), // old image to avoid regression in injector.
```

Comment:
Can you add a check at the beginning of the test to make sure that `DAPR_TEST_OLD_IMAGE` is defined?

Reply:
I can't because it does not work on Windows. Remember that 1.9.6 or 1.10.7 don't have the latest windows images.

Reply:
Makes sense